src/tab-panels.js

```diff
@@ -24,18 +36,18 @@ const TabPanels = ({
     ))
 
   return (
-    <>
-      <TabsComponent value={value} onChange={onTabChange}>
+    <div className={TabsProps.orientation === 'vertical' ? classes.vertical : null}>
```

Comment:
would this be effectively equivalent?

```suggestion
    <div className={classes[TabsProps.orientation]}>
```

Reply:
yes it would! 😃